lib/utils/useExternalErc20Awards.js

```diff
@@ -0,0 +1,86 @@
+import { batch, contract } from '@pooltogether/etherplex'
```

Comment:
There's a file in this PR called `lib/utils/fetchPrizePoolData.js` but it's empty. Delete?
<img width="486" alt="Screenshot 2020-12-03 at 13 39 33" src="https://user-images.githubusercontent.com/35717/101091405-fb130580-356c-11eb-9e53-6ce748bed3fb.png">